src/lints/enum_variant_missing.ron

```diff
@@ -56,4 +58,7 @@ SemverQuery(
     },
     error_message: "A publicly-visible enum has at least one variant that is no longer available under its prior name. It may have been renamed or removed entirely.",
     per_result_error_template: Some("variant {{enum_name}}::{{variant_name}}, previously in file {{span_filename}}:{{span_begin_line}}"),
+    witness: (
+        hint_template: r#"let witness = {{join "::" path}}::{{variant_name}}{{#if (eq kind "StructVariant")}}{...}{{/if}}{{#if (eq kind "TupleVariant")}}(...){{/if}};"#,
```

Comment:
To the best extent possible, let's try to output code that is analogous to how `cargo fmt` would represent it, so that we aid readability. Tuple variants get constructed as `Enum::Tuple(a, b)` but struct variants get constructed as `Enum::Struct { a, b }` — note the space before the curly brace.
```suggestion
        hint_template: r#"let witness = {{join "::" path}}::{{variant_name}}{{#if (eq kind "StructVariant")}} {...}{{/if}}{{#if (eq kind "TupleVariant")}}(...){{/if}};"#,
```